### PR TITLE
APIS-6482 Fix issue with fraud prevention nav link

### PR DESCRIPTION
**Pull request by petekirby-ee**

### CODE REVIEW

File: app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/fraudprevention/FraudPreventionNavLinkHelper.scala [hunk 1]
```diff
@@ -37,7 +37,7 @@ trait FraudPreventionNavLinkHelper {
     if (fraudPreventionConfig.enabled) {
       val apis            = fraudPreventionConfig.apisWithFraudPrevention
       val isProduction    = application.deployedTo == Environment.PRODUCTION
-      val shouldBeVisible = subscriptions.exists(x => apis.contains(x.serviceName) && x.subscribed && isProduction)
+      val shouldBeVisible = subscriptions.exists(x => apis.contains(x.serviceName.value) && x.subscribed && isProduction)
```
Comment (mattclark-zerogravit) [inline app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/fraudprevention/FraudPreventionNavLinkHelper.scala:40]: I think the list from config should be a list of serviceNames

Comment (petekirby-ee) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/fraudprevention/FraudPreventionNavLinkHelper.scala:40]: I agree, but how about merging this quickly to fix the issue in QA?

Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/fraudprevention/FraudPreventionNavLinkHelper.scala:40]: Turning a strong type to a string feels like the wrong fix.

Comment (AndySpaven) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/fraudprevention/FraudPreventionNavLinkHelper.scala:40]: Agreed it should be done right or we'll never do it - it's only QA

Comment (petekirby-ee) [reply app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/fraudprevention/FraudPreventionNavLinkHelper.scala]: Fixed

**Review by mattclark-zerogravit — APPROVED** [review]
👍

**Pull request merged**